basis/ui/tools/tools.factor

```diff
@@ -1,13 +1,16 @@
 ! Copyright (C) 2006, 2009 Slava Pestov.
 ! See http://factorcode.org/license.txt for BSD license.
-USING: memory system kernel vocabs.refresh ui.tools.operations
-ui.tools.listener ui.tools.browser ui.tools.common ui.tools.error-list
-ui.tools.walker ui.commands ui.gestures ui ui.private ;
+USING: kernel listener memory system ui ui.commands ui.gestures
+ui.private ui.tools.browser ui.tools.common ui.tools.error-list
+ui.tools.listener ui.tools.operations ui.tools.walker
+vocabs.refresh ;
 IN: ui.tools
 
 MAIN: listener-window
 
-\ refresh-all H{ { +nullary+ t } { +listener+ t } } define-command
+: refresh-all-cmd ( -- ) [ refresh-all ] with-ctrl-break ;
+
+\ refresh-all-cmd H{ { +nullary+ t } { +listener+ t } } define-command
```

Comment:
I'm not sure if that's a good idea, so I'll let someone with more knowledge of the Factor UI framework to reply.

My goal was to localize the changes and have as little side-effects to parts unknown to me as possible.